src/components/Buttons/ClipboardButton.tsx

```diff
@@ -92,3 +111,17 @@ export default styled(ClipboardButton)`
     color: ${({ theme }) => theme.font.primary};
   }
 `
+
+export default styled(ClipboardButton)`
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+
+  &:hover > ${CellClipboard} {
+    opacity: 1;
+  }
+
+  &:hover > ${CellChildren} {
+    -webkit-mask-image: linear-gradient(to right, rgba(0, 0, 0, 1) 50%, rgba(0, 0, 0, 0) calc(100% - 15px));
```

Comment:
`CellClipboard` and `CellChildren` must be defined before these template interpolations, otherwise an error occurs, so they appear before the export.